### DISCUSSION TITLE
Copilot Not Working in VS Code Despite Authentication

### DISCUSSION THREAD

**Discussion by agreea** [community/community#50848] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Here's what I see in VS code:
![image](https://user-images.githubusercontent.com/2617315/227318551-97668734-f81e-4849-93be-91c39d14179d.png)

The UI suggests:
- it's authenticated
- it's enabled

But it's not suggesting code.

Here's what I've tried so far:

[X] Reloading VSCode.
[X] Restarted Mac.
[X] Reinstalled both VSCode and Copilot.
[X] Resigned into my GitHub account in VS Code.

Note that it works on a fresh install of VS Code on a laptop that's never had it installed before, with me logged in to the same Github account.

Here's what I see on the latest install of the Github Copilot (v1.78.9758) in the output:

> [INFO] [codequote] [2023-03-22T17:26:53.245Z] CodeQuote disabled

The Electron developer console shows this output:

Extension 'github.copilot' wants API proposal 'interactive' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.

I tried [this solution](https://github.com/community/community/discussions/50305#discussioncomment-5348201) - and while the "CodeQuote disabled" error goes away, Copilot still doesn't auto-recommend.

Here are the Copilot diagnostics:
## Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Self Signed Certificates: disabled
- Send Restricted Telemetry: disabled

## Extensions

- Copilot Version: 1.78.9758
- Is `win-ca` installed?: true
- Is `mac-ca` installed?: true

## Node setup

- Number of root certificates: 131
- Operating system: Darwin
- Operating system version: 22.3.0
- Operating system architecture: arm64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 

Here's the VS Code Setup:
Version: 1.76.2 (Universal)
Commit: ee2b180d582a7f601fa6ecfdad8d9fd269ab1884
Date: 2023-03-14T17:54:09.061Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Darwin arm64 22.3.0
Sandboxed: No

**Comment by agreea** [comment, marked as answer]
Resolved: had to uninstall VS Code and delete its data ([according to these answers](https://stackoverflow.com/questions/42603103/how-to-completely-uninstall-vs-code-on-mac)), reinstall VS Code. Working now!

**Comment by khanhnd0408** [comment]
I had tried to uninstall vscode and delete vscode data at: `%APPDATA%\Code and %USERPROFILE%\.vscode .`, also extensions but it not working for me, any help :(
Copilot logs:
![image](https://user-images.githubusercontent.com/35426596/227837700-9859975a-5b3c-4151-8825-69fa00a07bd6.png)
Github  Authen logs:
![image](https://user-images.githubusercontent.com/35426596/227837760-2cf841d8-c4ab-4822-a154-d796825a78e0.png)
